components/prism-javascript.js

```diff
@@ -9,6 +9,24 @@ Prism.languages.javascript = Prism.languages.extend('clike', {
 Prism.languages.insertBefore('javascript', 'keyword', {
 	'regex': {
 		pattern: /((?:^|[^$\w\xA0-\uFFFF."'\])\s])\s*)\/(\[[^\]\r\n]+]|\\.|[^/\\\[\r\n])+\/[gimyu]{0,5}(?=\s*($|[\r\n,.;})\]]))/,
+		inside: {
+			'regex-flags': {
+				pattern: /[gimyu]{1,5}$/,
+				alias: 'keyword'
+			},
+			'regex-group': {
+				pattern: /\[(?:\\\\|\\\]|[^\]])*\]/,
+				alias: 'escape'
+			},
+			'regex-escape': {
+				pattern: /\\./,
+				alias: 'escape'
+			},
+			'regex-punctuation': {
+				pattern: /[|^$*+(){}]|(?:\?(?![:!=<]))/,
+				alias: 'operator'
```

Comment:
`regex-punctuation`, alias `operator`. What?
I know you did it for the highlighting, but this is somewhat confusing...

Reply:
I admit that "punctuation" and "operator" is also a bad name, maybe "regex-token" is better?

And I found that VSCode highlights quantifiers (such as `{1,2}`) just like the other "punctuations", I'll add this situations later.

Removing the alias and changing the name is easy, but the matched characters will not have special color unless we modify the themes...

Reply:
Yeah, editing the themes is a little overkill...